README.rst

```diff
@@ -22,7 +22,7 @@ STPSF: Simulated Point Spread Functions for the James Webb and Nancy Grace Roman
    :target: http://ascl.net/1504.007
 
 
-**ADVISORY:  STPSF IS CURRENTLY BEING MIGRATED FROM WEBBPSF - THIS REPOSITORY IS NOT READY FOR USE**
+**STPSF HAS BEEN MIGRATED FROM WEBBPSF**
```

Comment:
Do we want to add STPSF has been migrated from webbpsf 1.5.0, so people know when was the cut? 
This is very minor.

Reply:
I think because its all in the release notes its not necessary, however if you disagree I can go back and re-do